### ISSUE TITLE
make' issues :: libtool (wrong version) && missing m4 directory

### ISSUE THREAD

**Issue by turbopascalwiz** [slacka/WoeUSB#117]
It would seem there is a missing directory 'm4' in the root directory, as well as the person/... being obligated to run Debian's version of libtool?

-- built using './configure' & 'make'

here is the output for both errors:

(missing m4 directory)
```bash
CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/sh /usr/src/GIT/WoeUSB/missing --run aclocal-1.11 -I m4
aclocal-1.11: couldn't open directory 'm4': No such file or directory
make: *** [Makefile:318: aclocal.m4] Error 1
```

(wrong libtool version)
```bash
...
checking for wxWidgets static library... no
configure: creating ./config.status

####################################

The package is now configured. You should type make !
    DEBUG MODE          : disable.
    TARGET PLATFORM     : linux.
    INSTALLATION PREFIX : /usr/local.

####################################
...
/bin/sh ../libtool --tag=CXX   --mode=link g++  -s -O1  -Wall -Wclobbered -Wempty-body -Wmissing-field-initializers -Wsign-compare -Wtype-limits -Wuninitialized   -o woeusbgui MainPanel.o App.o findFile.o MyException.o strWxStdConv.o MainFrame.o DialogAbout.o processManager.o -L/usr/lib64 -pthread   -lwx_gtk2u_xrc-3.0 -lwx_gtk2u_html-3.0 -lwx_gtk2u_qa-3.0 -lwx_gtk2u_adv-3.0 -lwx_gtk2u_core-3.0 -lwx_baseu_xml-3.0 -lwx_baseu_net-3.0 -lwx_baseu-3.0  -lstdc++ 
libtool: Version mismatch error.  This is libtool 2.4.2 Debian-2.4.2-1ubuntu1, but the
libtool: definition of this LT_INIT comes from libtool 2.4.6.
libtool: You should recreate aclocal.m4 with macros from libtool 2.4.2 Debian-2.4.2-1ubuntu1
libtool: and run autoconf again.
make[3]: *** [Makefile:444: woeusbgui] Error 63
make[3]: Leaving directory '/usr/src/GIT/WoeUSB/src'
make[2]: *** [Makefile:573: all-recursive] Error 1
make[2]: Leaving directory '/usr/src/GIT/WoeUSB/src'
make[1]: *** [Makefile:347: all] Error 2
make[1]: Leaving directory '/usr/src/GIT/WoeUSB/src'
make: *** [Makefile:380: all-recursive] Error 1


```

**Comment by Lin-Buo-Ren** [comment]
Have you run `autoreconf --force --install` ?  If you run this command will the issue still can be reproduced?

**Comment by turbopascalwiz** [comment]
works like a charm, thank you for your attention

**Comment by turbopascalwiz** [comment]
seems like it still complains about the missing directory m4. but autoreconf seemed to take care of the libtool issue. Is there supposed to be a m4 directory in the repo? That seems to be the only fix required for 'this issue'.

**Comment by salvadorp2001** [comment]
It needs at least an empty "m4" folder, but I made a fork and tried to add an empty one and well, git seems to not like empty folders. The fix for now is just to create an empty folder.

**Comment by turbopascalwiz** [comment]
it is possible to create an 'empty folder' by using a .gitignore file, but if you can figure it in to a configuration script (autoconf) like so: 

```bash
build:
    test -d ${srcdir}/m4 || mkdir ${srcdir}/m4
```

**Comment by salvadorp2001** [comment]
Nvm, it doesn't actually need m4, it was just a flag in Makefile.am that was left over from who knows where

**Comment by salvadorp2001** [comment]
Fixed by commit #118, could someone with write permission close this?

**Comment by Lin-Buo-Ren** [comment]
Thanks.  Closing.
